test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -556,5 +559,184 @@ abstract class LorisIntegrationTest extends TestCase
             );
         }
     }
+    /**
+     * Get the content of a React Element
+     *
+     * @param  string $ui the location of this ui (css selector)
+     * @return string
+     */
+    function getReactElementContent($ui)
+    {
+        $attempts = 0;
+        $err = null;
+        do {
+            try {
+                return $this->webDriver->executescript(
+                    "return document.querySelector('$ui').textContent"
+                );
+            } catch (Exception $e) {
+         //       echo 'Caught exception: ',  $e->getMessage(), "\n";
+                        $err = $e;
+                        $attempts++;
+                        sleep(1);
+                        continue;
+            }
+        } while($attempts < self::NUM_OF_ATTEMPTS);
+        if ($err && $attempts > 4) {
+           echo 'Caught exception: ',  $err->getMessage(), "\n";
```

Comment:
Shouldn't this fail the test with the error instead of just printing the message to stdout?